TotalSegmentator/Resources/UI/TotalSegmentator.ui

```diff
@@ -217,6 +217,23 @@
         </property>
        </widget>
       </item>
+      <item row="1" column="1">
+       <widget class="QCheckBox" name="pullMasterBranchCheckBox">
+        <property name="toolTip">
+         <string>Use TotalSegmentator master branch (latest changes)  during forced reinstall. </string>
+        </property>
+        <property name="text">
+         <string/>
+        </property>
+       </widget>
+      </item>
+      <item row="1" column="0">
+       <widget class="QLabel" name="label_2">
+        <property name="text">
+         <string>Use master branch</string>
```

Comment:
It is OK to use such very technical jargon in explanations in tooltips but on the GUI try to use words that you think your colleagues would understand. Something like "Use latest development version" could be good.